src/components/axes/axis.ts

```diff
@@ -54,12 +55,13 @@ module Plottable {
 
       this.formatter(formatter);
 
-      this._scale.broadcaster.registerListener(this, () => this._rescale());
+      this._rescaleCallback = (scale) => this._rescale();
```

Comment:
Or does it not? Apparently I can omit the (scale) argument, but that seems weird. Should I remove it where irrelevant?